library/python/src/logging.cc

```diff
@@ -0,0 +1,32 @@
+#include "pywavemap/logging.h"
+
+#include <glog/logging.h>
+#include <nanobind/stl/string.h>
+#include <wavemap/core/utils/logging_level.h>
+
+using namespace nb::literals;  // NOLINT
+
+namespace wavemap {
+void add_logging_module(nb::module_& m_logging) {
+  // Initialize GLOG
+  google::InitGoogleLogging("pywavemap");
+  google::InstallFailureSignalHandler();
+  FLAGS_alsologtostderr = true;
+  FLAGS_colorlogtostderr = true;
+  FLAGS_log_prefix = false;
+
+  // Methods to configure GLOG
+  m_logging.def(
+      "set_level",
```

Comment:
This turns into `wave.logging.set_level("error")` at the end?

Reply:
Yes exactly